sphinx/ext/doctest.py

```diff
@@ -330,7 +330,7 @@ def _warn_out(self, text):
             logger.warning(text)
         else:
             logger.info(text, nonl=True)
-        if isinstance(text, binary_type):
+        if isinstance(text, bytes):
```

Comment:
I don't know that doctest gives bytes object or not. But I suppose this is not needed. binary object is mainly used on IO boundary.